### DISCUSSION TITLE
How add rows/columns?

### DISCUSSION THREAD

**Discussion by mamcx** [sevmeyer/chrumm-keyboard#9] [Q&A]
What steps are involved in adding more rows/columns to this board? I wanna[ revamp mine](https://www.reddit.com/r/MechanicalKeyboards/comments/9wjpg0/finished_ergo_keyboard/) and this looks closer to my ideal...

**Comment by sevmeyer** [comment]
The layout is defined in `body/chrumm.json`.
More information about the parameters can be found in `body/README.md`.
Also check `images/layout.svg` for a visual representation of the default json parameters.
